### DISCUSSION TITLE
Use Markor (md) Files with Windows application

### DISCUSSION THREAD

**Discussion by t-arn** [gsantner/markor#1568] [Q&A] [answered]
I backup my Markor data to my Windows PC.

Is there a Windows application with which I could use my Markor data (markdown) without conversion on my PC?

**Comment by aubreyz** [comment, marked as answer]
Obsidian

**Reply by harshad1** [reply]
Any good text editor will support unix line endings (which markor uses). For example, notepad++

**Reply by t-arn** [reply]
Obsidian is exactly what I looked for.
Thanks for the hint!

**Reply by Yuzgaf33tref6** [reply]
Thanks！！！
